ui/tests/unit/unload-test.js

```diff
@@ -70,7 +70,7 @@ module('Unit | Model | unloadAll works as expected', function (hooks) {
 
     assert.strictEqual(this.store.peekAll('company').length, 0, 'peekAll 0 - companies unloaded');
     assert.strictEqual(
-      this.store.peekAll('company').toArray().length,
+      this.store.peekAll('company').slice().length,
```

Comment:
Is this because `.length` can't be called on the record array returned?

Reply:
You can, the line above is checking length on `peekAll`! This test was written partially to make sure that the store methods used in lazy pagination were working correctly, and since we still are using `slice` there I kept this in.

Reply:
Ah! Okay cool - thanks for explaining